backend/pkg/sqlconnect/sql-connector.go

```diff
@@ -34,24 +34,54 @@ type SqlDBTX interface {
 	BeginTx(context.Context, *sql.TxOptions) (*sql.Tx, error)
 }
 
+type SqlConnectorOption func(*sqlConnectorOptions)
+
+type sqlConnectorOptions struct {
+	mysqlDisableParseTime bool
+	postgresDriver        *string
+}
+
+// WithMysqlDisableParseTime disables MySQL time parsing
+func WithMysqlDisableParseTime(disable bool) SqlConnectorOption {
+	return func(opts *sqlConnectorOptions) {
+		opts.mysqlDisableParseTime = disable
+	}
+}
+
+// WithPostgresDriver overrides default postgres driver
+func WithPostgresDriver(postgresDriver string) SqlConnectorOption {
```

Comment:
Similar thing here. Might be clearer for us if we for instance default to pgx, to have an option simply called: `WithDefaultPostgresDriver()` the just does `opts.postgresDriver = "postgres"`

There are few options here so we can make it brain dead simple by just having these functions that fully encapsulate the intent.